test/sample_search_api_server_test.py

```diff
@@ -59,6 +59,9 @@ def setUpClass(cls):
             {'id': 'b969c622-ea18-4dda-9943-bf1692e526dd', 'version': 1}
         ]
 
+        # corresponding sample set object ref for the above samples
+        cls.sampleset_object_ref = '64822/2/1'
```

Comment:
This is okay for now but we should move these tests data sets into the Examples workspace so that the tests work for other users and across deploys.